src/termdict/merger.rs

```diff
@@ -39,15 +39,20 @@ impl<'a> TermMerger<'a> {
         }
     }
 
+    /// Iterator over (segment ordinal, [TermOrdinal]) sorted by segment ordinal
+    ///
+    /// This method may be called
+    /// if [Self::advance] has been called before
+    /// and [true] was returned.
     pub fn matching_segments<'b: 'a>(&'b self) -> impl 'b + Iterator<Item = (usize, TermOrdinal)> {
         self.current_segment_and_term_ordinals
             .iter()
             .map(|iv| (iv.index, iv.value))
     }
 
     /// Advance the term iterator to the next term.
-    /// Returns true if there is indeed another term
-    /// False if there is none.
+    /// Returns [true] if there is indeed another term
```

Comment:
I think the brackets in rustdoc are here to generate links. `true` is what you want if you just want a monospace font.
You can combine both [`TermDictionary::advance`]

Reply:
yeah, it's a pity these linked are color coded to the types :)